RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate method getSettings to typescript

## Proposed changes

Migrate method getSettings to typescript

## Issue(s)	


## How to test or reproduce

Log into the app and see if the settings are being saved correctly.
Try updating some settings on the server and see if the app will replicate the change.

## Screenshots

## Types of changes




## Checklist



## Further comments
